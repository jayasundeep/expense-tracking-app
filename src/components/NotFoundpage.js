import React from 'react';

const NotFoundpage = () => (
    <div>
        Page Not Found 404! WTF?
    </div>
);

export default NotFoundpage;