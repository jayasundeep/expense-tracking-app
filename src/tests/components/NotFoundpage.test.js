import React from 'react';
import { shallow } from 'enzyme';
import NotFoundpage from '../../components/NotFoundpage';

test('should render Header component', () => {
    const wrapper = shallow(<NotFoundpage />);
    expect(wrapper).toMatchSnapshot();
});