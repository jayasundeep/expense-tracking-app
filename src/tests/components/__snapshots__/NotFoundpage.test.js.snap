// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
<div>
  Page Not Found 404!
</div>
`;
